lib/srv/db/common/engines.go

```diff
@@ -130,3 +148,18 @@ func (c *EngineConfig) CheckAndSetDefaults() error {
 	}
 	return nil
 }
+
+func (c *EngineConfig) ObserveConnectionSetupTime() func() {
+	start := c.Clock.Now()
+	return func() {
+		connectionSetupTime.WithLabelValues(teleport.ComponentDatabase, c.Name).Observe(c.Clock.Since(start).Seconds())
```

Comment:
solely database name label won't  give us to much information especially for cloud customers. Could we add the database type and distinguish the real type like RDS slefhosted, AWS hosted  ?

Reply:
`name` is the name of the engine, not the database. Do you think we should add the actual database name here too?